### PR TITLE
DEVEL-884/unify container width

### PR DESCRIPTION
**Pull request by Simply007**
### Motivation

DEVEL-884

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

If manual testing is required, what are the steps?

### CODE REVIEW

File: components/shared/ui/footer.tsx [hunk 1]
```diff
@@ -10,8 +10,10 @@ export const Footer: FC<Props> = props => {
   const siteCodename = useSiteCodename();
 
   return (
-    <footer className={`w-full grow-0 h-16 px-10 flex flex-auto shrink-0 justify-end items-center ${mainColorBgClass[siteCodename]} ${props.className ?? ""}`}>
-      <p>Ficto Healthtech inc.</p>
+    <footer className={`${mainColorBgClass[siteCodename]} w-screen`}>
+      <div className="flex gap-4 items-center mx-auto max-w-screen-xl md:h-16">
```
Comment (IvanKiral) [inline components/shared/ui/footer.tsx:14]:
```suggestion
      <div className="flex items-center mx-auto max-w-screen-xl md:h-16">
```

I think gap is not necessary as there is only one element :)

Comment (IvanKiral) [reply components/shared/ui/footer.tsx:14]: image

Let's make it a little bit nice on smaller than 'xl' breakpoints probably add padding px-8 (number can be different) and xl:px-0?

Comment (IvanKiral) [inline components/shared/ui/footer.tsx:14]: Why only set height on md? don't we want it to have some space in mobile devices?

image

Comment (Simply007) [inline components/shared/ui/footer.tsx]:
```suggestion
      <div className="flex gap-4 items-center mx-auto max-w-screen-xl h-16">
```

Comment (Simply007) [inline components/shared/ui/footer.tsx]:
```suggestion
      <div className="flex gap-4 items-center mx-auto max-w-screen-xl h-16">
```

Comment (Simply007) [reply components/shared/ui/footer.tsx]: Will do

Comment (Simply007) [reply components/shared/ui/footer.tsx]: 9117424: set height for all breakpoints

Comment (Simply007) [reply components/shared/ui/footer.tsx]: 35c2922: add x padding fopr footer content

**Review by IvanKiral — APPROVED** [review]

**Pull request merged**